src/plugins/plugin.legend.js

```diff
@@ -459,58 +465,75 @@ var Legend = Element.extend({
 		}
 	},
 
+	/**
+	 * Gets the legend item, if any, at a given x,y position.
+	 * @private
+	 * @param {number} x - The x coordinate to check
+	 * @param {number} y - The y coordindate to check
+	 * @return {?Object} - The legend item found
+	 */
+	getLegendItemAt: function(x, y) {
```

Comment:
This method should be explicitly private (prefixed by `_`) and would remove the jsdoc comment which is a bit useless and would keep only:

```javascript
/**
 * @private
 */
_getLegendItemAt: function(x, y) {
```